# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1260223496 28800
#      Mon Dec 07 14:04:56 2009 -0800
# Node ID 519de8bb56ab46e93b266dca7209565c3f95af0c
# Parent  ac4b97c6bf8b2b51d8dbc5262dfd025538627ccd
Add CLEANFILES and MAINTAINERCLEANFILES objects to properly clean refcard directory

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,13 @@
+2009-12-07  Rik <octave@nomad.inbox5.com>
+
+	* refcard/Makefile.am: Add CLEANFILES and MAINTAINERCLEANFILES objects to
+	properly clean directory.
+
 2009-12-05  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/Makefile.am: Set dependency of *.texi files on
 	munge-texi source code rather than executable.
 
 2009-12-04  Rik <octave@nomad.inbox5.com>
 
 	* interpreter/images.awk: Revert to using Ghostscript for documentation 
diff --git a/doc/refcard/Makefile.am b/doc/refcard/Makefile.am
--- a/doc/refcard/Makefile.am
+++ b/doc/refcard/Makefile.am
@@ -35,17 +35,22 @@ FORMATTED = \
   refcard-legal.dvi \
   refcard-legal.ps \
   refcard-legal.pdf \
   refcard-letter.dvi \
   refcard-letter.ps \
   refcard-letter.pdf
 
 EXTRA_DIST = \
-  $(TEX) $(FORMATTED)
+  $(TEX) \
+  $(FORMATTED)
+
+CLEANFILES = refcard-a4.log refcard-legal.log refcard-letter.log
+
+MAINTAINERCLEANFILES = $(FORMATTED)
 
 all-local: $(FORMATTED)
 
 refcard-a4.dvi: refcard-a4.tex refcard.tex
 	-TEXINPUTS="$(srcdir):$(TEXINPUTS):" tex $<
 
 refcard-a4.ps: refcard-a4.dvi
 	-dvips -T 297mm,210mm -o $@ $<
